File: app/src/main/java/com/example/android/miniweather/Manager/NavigationManager.java
package com.example.android.miniweather.Manager;

import android.app.Fragment;
import android.app.FragmentManager;


/**
 * Created by matev on 2017. 11. 30..
 */

public class NavigationManager {

    public static void moveToScreen (FragmentManager fragmentManager, Fragment fragment) {
        fragmentManager.beginTransaction()
                .replace(android.R.id.content, fragment)
                .addToBackStack(null)
                .commit();
    }

    public static void moveToFirstScreen (FragmentManager fragmentManager, Fragment fragment) {
        fragmentManager.beginTransaction()
                .replace(android.R.id.content, fragment)
                .commit();
    }
}

File: app/src/main/java/com/example/android/miniweather/Manager/NavigationManager.kt
package com.example.android.miniweather.Manager

import android.app.Fragment
import android.app.FragmentManager


/**
 * Created by matev on 2017. 11. 30..
 */

object NavigationManager {

    @JvmStatic
    fun moveToScreen(fragmentManager: FragmentManager, fragment: Fragment) {
        fragmentManager.beginTransaction()
                .replace(android.R.id.content, fragment)
                .addToBackStack(null)
                .commit()
    }

    @JvmStatic
    fun moveToFirstScreen(fragmentManager: FragmentManager, fragment: Fragment) {
        fragmentManager.beginTransaction()
                .replace(android.R.id.content, fragment)
                .commit()
    }
}
